docs/usage/models.md

```diff
@@ -871,19 +860,20 @@ print([pet for pet in pets])
 
 ## Faux immutability
 
-Models can be configured to be immutable via `allow_mutation = False`. When this is set, attempting to change the
+Models can be configured to be immutable via `frozen = True`. When this is set, attempting to change the
```

Comment:
Can we add a note about the change from `allow_mutation` to `frozen`?

Reply:
added